### DISCUSSION TITLE
Wallet balance not changing on deposit

### DISCUSSION THREAD

**Discussion by rrutwik** [bavix/laravel-wallet#728] [Q&A]
I am using 
$this->deposit($amount, $meta);
to deposit amount in a user wallet.
In logs, i see balance changing.. but it does not change in database.
Transaction is updated fine, in database

**Comment by rez1dent3** [comment]
Does the update happen in a transaction?

If yes, then you will have to replace laravel transactions with package transactions: https://bavix.github.io/laravel-wallet/#/transaction?id=if-you-are-using-a-version-below-96-then-the-following-is-relevant-for-you

If you do not want or cannot use package transactions, then you need to upgrade your wallet to version 9.6+, because. for the wallet, changes were made to the framework itself: https://github.com/laravel/framework/pull/44608
